content/windows/docs/hardening.md

```diff
@@ -1,37 +1,20 @@
-
-This guide focus specifically on Windows nodes running on Amazon Elastic Kubernetes Service (EKS).
+Microsoft offers a range of tools like [Microsoft Security Compliance Toolkit](https://www.microsoft.com/en-us/download/details.aspx?id=55319) and [Security Baselines](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-security-baselines) that helps you to achieve hardening based on your security policies needs. [CIS Benchmarks](https://learn.cisecurity.org/benchmarks?_gl=1*eoog69*_ga*MTgzOTM2NDE0My4xNzA0NDgwNTcy*_ga_3FW1B1JC98*MTcwNDQ4MDU3MS4xLjAuMTcwNDQ4MDU3MS4wLjAuMA..*_ga_N70Z2MKMD7*MTcwNDQ4MDU3MS4xLjAuMTcwNDQ4MDU3MS42MC4wLjA.) are also available and should be implemented on top of an Amazon EKS Optimized Windows AMI for production environments.
 
 ## Reducing attack surface with Windows Server Core
 
-Windows Server Core is a minimal installation option that is available as part of the [EKS Optimized Windows AMI](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-windows-ami.html). Deploying Windows Server Core has a couple benefits. First, it has a relatively small disk footprint being 6GB on Server Core against 10GB on Windows Server with Desktop experience. Second, it has smaller attack surface because of its smaller code base.
-
-You can specify the Server Core EKS Optimized AMI for Windows when you deploy your nodes through `eksctl` or Cloudformation.
-
-The example below is an eksctl manifest for a Windows node group based on Windows Server Core 2004:
-
-```yaml
-nodeGroups:
-- name: windows-ng
-  instanceType: c5.xlarge
-  minSize: 1
-  volumeSize: 50
-  amiFamily: WindowsServer2019CoreContainer
-  ssh:
-    allow: false
-```
+Windows Server Core is a minimal installation option that is available as part of the [EKS Optimized Windows AMI](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-windows-ami.html). Deploying Windows Server Core has a couple of benefits. First, it has a relatively small disk footprint, being 6GB on Server Core against 10GB on Windows Server with Desktop experience. Second, it has a smaller attack surface because of its smaller code base and available APIs.
 
-The amiFamily name conventions can be found on the [eksctl official documentation.](https://eksctl.io/usage/custom-ami-support/)
+AWS provides customers with new Amazon EKS Optimized Windows AMIs every month, containing the latest Microsoft security patches, regardless of the Amazon EKS-supported version. As a best practice, Windows worker nodes must be replaced with new ones based on the latest Amazon EKS-optimized AMI. Any node running for more than 45 days without an update in place or node replacement lacks security best practices.
```

Comment:
Is this really true? Do customers who run Windows worker nodes replace their instances or do they patch or some combination of both?